src/WelcomeForm/WelcomeForm.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+
+class WelcomeForm extends Component {
+  constructor() {
+    super();
+    this.state = {};
+  }
+
+  render() {
+    return(
+      <form>
+        <label>Name</label>
+        <input placeHolder='Galactic Identifier'></input>
+        <label>Quote</label>
+        <input placeHolder='Do or do not, there is no try.'></input>
+        <label>Experience Level</label>
+        <select>
+          <option value='padawan'>Padawan</option>
+          <option value='jediknight'>Jedi Knight</option>
+          <option value='jedimaster'>Jedi Master</option>
+        </select>
+        <button>Enter, you must!</button>
```

Comment:
Great start for the drop down. Love the names to stay with the theme! Something we'll want to keep in mind is keeping tracked of the 'selected' option within the dropdown - might need to remind myself how to do that.